app/src/controllers/uniqueErrors/clusterItems/reducer.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  FETCH_CLUSTER_ITEMS_START,
+  FETCH_CLUSTER_ITEMS_SUCCESS,
+  CLEAR_CLUSTER_ITEMS,
+  TOGGLE_CLUSTER_ITEMS,
+} from './constants';
+
+export const clusterItemsReducer = (state = {}, action) => {
+  switch (action.type) {
+    case FETCH_CLUSTER_ITEMS_START: {
+      const {
+        payload: { id },
+        refresh = false,
```

Comment:
Where you provide this property to the action?

Reply:
It was an idea that is no longer used